templates/ray_serve_llm/README.md

```diff
@@ -0,0 +1,160 @@
+# Deploy, configure, and serve LLMs 
+
+**⏱️ Time to complete**: 10 min
+
+This template comes with a library for serving OSS LLMs on Anyscale called [Ray Serve LLM](https://docs.ray.io/en/releases-2.43.0/serve/llm/overview.html).
+
+Ray Serve LLM provides a number of features that simplifies LLM development, including:
+- An extensive suite of pre-configured open source LLMs.
+- An OpenAI-compatible REST API.
+
+As well as operational features to efficiently scale LLM apps:
+- Optimizations such as continuous batching, quantization and streaming.
+- Production-grade autoscaling support, including scale-to-zero.
+- Native multi-GPU & multi-node model deployments.
+
+This template explains how to set up, run, and query LLMs with Ray Serve LLM.
+
+To learn more about Ray Serve LLM, check out [the docs](https://docs.ray.io/en/releases-2.43.0/serve/llm/overview.html). If you're interested in fine-tuning, check out the [fine-tuning template](https://console.anyscale.com/v2/template-preview/finetuning_llms_v2).
+
+**Note**: This guide is hosted within an Anyscale workspace, which provides easy access to compute resources. Check out the [Introduction to Workspaces](https://docs.anyscale.com/examples/intro-workspaces/) template for more details.
+
+## Step 1 - Generate a LLM config
+
+Ray Serve LLM takes in a config that specifies the model you wish to serve, as well as its settings.
+
+Use the Ray Serve LLM CLI in the workspace terminal to generate the config:
+
+```bash
+python -m ray.serve.llm.gen_config
+```
+
+**Note:** This command requires interactive inputs and should be executed directly in the terminal, not within a Jupyter notebook cell.
+
+This command lets you pick from a common set of OSS LLMs and helps you configure them. You can tune settings like GPU type, tensor parallelism, and autoscaling parameters.
+
+Please note that if you're configuring a model whose architecture is different from the provided list of models, we recommend that you closely review the generated model config file to provide the correct values.
+
+This command generates 2 files - an LLM config file (saved in `model_config/`) and a Ray Serve config file (`serve_TIMESTAMP.yaml`) that you can reference and re-run in the future.
```

Comment:
so in here let's mention two things: 

1. They should read the generated config and how it would look like. 
2. They should read vllm (or the related engine) documentation to learn about engine level argument that they can specify through enigne_kwargs.